plugin.xml

```diff
@@ -132,14 +132,14 @@
 				<source url="https://cdn.cocoapods.org/"/>
 			</config>
 			<pods use-frameworks="true">
-				<pod name="Firebase/Core" spec="6.28.0"/>
-				<pod name="Firebase/Auth" spec="6.28.0"/>
-				<pod name="Firebase/Messaging" spec="6.28.0"/>
-				<pod name="Firebase/Performance" spec="6.28.0"/>
-				<pod name="Firebase/RemoteConfig" spec="6.28.0"/>
-				<pod name="Firebase/InAppMessaging" spec="6.28.0"/>
-				<pod name="FirebaseFirestore" git="https://github.com/invertase/firestore-ios-sdk-frameworks.git" tag="6.28.0"/>
-				<pod name="Firebase/Crashlytics" spec="6.28.0"/>
+				<pod name="Firebase/Core" spec="6.28.1"/>
+				<pod name="Firebase/Auth" spec="6.28.1"/>
+				<pod name="Firebase/Messaging" spec="6.28.1"/>
+				<pod name="Firebase/Performance" spec="6.28.1"/>
+				<pod name="Firebase/RemoteConfig" spec="6.28.1"/>
+				<pod name="Firebase/InAppMessaging" spec="6.28.1"/>
+				<pod name="Firebase/Firestore" spec="6.28.1"/>
```

Comment:
This pre-compiled version needs to be kept as it significantly reduces build times

Reply:
> This pre-compiled version of Firestore needs to be kept as it significantly reduces build times

The pre-compiler firestore version freeze the ios compilation on my side.

Ionic:

Ionic CLI : 6.10.1 (/usr/local/lib/node_modules/@ionic/cli)
Ionic Framework : @ionic/angular 5.2.3
@angular-devkit/build-angular : 0.1000.3
@angular-devkit/schematics : 10.0.3
@angular/cli : 10.0.3
@ionic/angular-toolkit : 2.2.0

Cordova:

Cordova CLI : 9.0.0 (cordova-lib@9.0.1)
Cordova Platforms : android 8.1.0, electron 1.1.1, ios 5.1.1
Cordova Plugins : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 8 other plugins)

Utility:

cordova-res (update available: 0.15.1) : 0.14.0
native-run : 1.0.0

System:

ios-deploy : 1.10.0
ios-sim : 8.0.2
NodeJS : v12.16.1 (/usr/local/bin/node)
npm : 6.13.4
OS : macOS Catalina
Xcode : Xcode 11.6 Build version 11E708

Reply:
The [example project](https://github.com/dpa99c/cordova-plugin-firebasex-test) builds fine for me with `cordova-plugin-firebasex@10.1.2` in Xcode 11.5 and Xcode 12.0 Beta 2.

Reply:
I follow the readme : 
```
git clone https://github.com/dpa99c/cordova-plugin-firebasex-test.git && cd cordova-plugin-firebasex-test
npm install

cordova platform add ios
cordova run ios
```

I obtain a build failed on the same step i'm freeze on my project : 
```
PhaseScriptExecution [CP]\ Copy\ Pods\ Resources /Users/damienblanchet/Library/Developer/Xcode/DerivedData/FirebaseTest-cqjrdjtdzlwcmiehwgvhhflbbajx/Build/Intermediates.noindex/FirebaseTest.build/Debug-iphonesimulator/FirebaseTest.build/Script-71CBF5A57D83BC59C2384146.sh (in target 'FirebaseTest' from project 'FirebaseTest')
    cd /Users/damienblanchet/Sources/cordova-plugin-firebasex-test/platforms/ios
    /bin/sh -c /Users/damienblanchet/Library/Developer/Xcode/DerivedData/FirebaseTest-cqjrdjtdzlwcmiehwgvhhflbbajx/Build/Intermediates.noindex/FirebaseTest.build/Debug-iphonesimulator/FirebaseTest.build/Script-71CBF5A57D83BC59C2384146.sh
/Users/damienblanchet/Sources/cordova-plugin-firebasex-test/platforms/ios/Pods/FirebaseFirestore/FirebaseFirestore/Resources/gRPCCertificates-Cpp.bundle
error: Resource "/Users/damienblanchet/Library/Developer/Xcode/DerivedData/FirebaseTest-cqjrdjtdzlwcmiehwgvhhflbbajx/Build/Products/Debug-iphonesimulator/FirebaseInAppMessaging/InAppMessagingDisplayResources.bundle" not found. Run 'pod install' to update the copy resources script.

** BUILD FAILED **
```

Reply:
https://github.com/dpa99c/cordova-plugin-firebasex#cordova-cli-builds

Reply:
You are right, I applied the modifications of this paragraph and I got a build success. I will investigate on my side to try to identify the issue